test/models/post_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
```

Comment:
Let's add a test for when a post is deleted comments and likes of that post should also be deleted.